sdk/core/amqp-common/.npmignore

```diff
@@ -0,0 +1,51 @@
+# git #
```

Comment:
We're using package.json files[] array field to specify the files that should be a part of the generated tarball. And we should not use .npm ignore. You can refer #2306 issue. You can do it as part of this PR or another PR for this. But before we release amqp-common